bfcfg

```diff
@@ -197,6 +198,219 @@ mfg_cfg()
   mfg_lock
 }
 
+add_trailing_null()
+{
+  for ((i = 0; i<${1}; i++))
+  do
+    printf '\0' >> ${2} 2>>${log_file}
+  done
+}
+
+mfg_ext_cfg()
+{
+  local tmp_file=/tmp/.bfcfg-mfg-ext-data
+  local MKCAP=/usr/lib/firmware/mellanox/boot/capsule/scripts/mlx-mkcap
+  local MFGEXT_CAPSULE=/tmp/MfgExtCapsule
+  local BFREC=/usr/bin/bfrec
```

Comment:
This command must be available, so no need to specify the full path

Reply:
Ack